tests/Backend/Workspaces/WorkspacesUnloadTest.php

```diff
@@ -38,45 +39,59 @@ public function testCreateTableFromWorkspace()
 
         $connection = $workspace['connection'];
 
-        $snowflakeConnection = new Connection([
-            'host' => $connection['host'],
-            'database' => $connection['database'],
-            'warehouse' => $connection['warehouse'],
-            'user' => $connection['user'],
-            'password' => $connection['password'],
-        ]);
-        $snowflakeConnection->query("USE SCHEMA " . $snowflakeConnection->quoteIdentifier($connection['schema']));
+        if ($connection['backend'] === parent::BACKEND_SNOWFLAKE) {
+            $db = new Connection([
+                'host' => $connection['host'],
+                'database' => $connection['database'],
+                'warehouse' => $connection['warehouse'],
+                'user' => $connection['user'],
+                'password' => $connection['password'],
+            ]);
+            // Set the session to use the workspace schema
+            $db->query("USE SCHEMA " . $db->quoteIdentifier($connection['schema']));
+
+        } else if ($connection['backend'] == parent::BACKEND_REDSHIFT) {
+            $db = new \PDO(
+                "pgsql:dbname={$connection['database']};port=5439;host=" . $connection['host'],
+                $connection['user'],
+                $connection['password']
+            );
+            //Redshift workspace user is auto-set to use correct workspace schema
+
+        } else {
+            throw new Exception("Backend not supported for workspaces");
+        }
 
-        $snowflakeConnection->query("create table \"test.Languages3\" (
+        $db->query("create table \"test.languages3\" (
 			\"Id\" integer not null,
 			\"Name\" varchar not null
 		);");
-        $snowflakeConnection->query("insert into \"test.Languages3\" (\"Id\", \"Name\") values (1, 'cz'), (2, 'en');");
-
+        $db->query("insert into \"test.languages3\" (\"Id\", \"Name\") values (1, 'cz'), (2, 'en');");
 
         // create table from workspace
         $tableId = $this->_client->createTableAsyncDirect($this->getTestBucketId(self::STAGE_IN), array(
-            'name' => 'languages',
+            'name' => 'languages3',
             'dataWorkspaceId' => $workspace['id'],
-            'dataTableName' => 'test.Languages3',
```

Comment:
@pivnicek why did yo changed the case? This is very important for testing varios edge cases.